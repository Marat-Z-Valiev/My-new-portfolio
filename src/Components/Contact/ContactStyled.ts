import styled from "styled-components";

const ConactStyled = styled.div`
	#contact {
		background-color: #ffffff;
		width: 100%;
		padding-top: 40px;
		padding-bottom: 40px;
	}
	h1 {
		text-align: center;
	}

	.links {
		display: flex;
		flex-direction: row;
		justify-content: center;

		.socialLinks {
			padding: 10px;
			background: #25242e;
			margin-right: 10px;

			i {
				color: #ffffff;
				transition: color ease-in 0.3s;

				&:hover {
					color: #077793;
				}
			}
		}
	}
`;

export default ConactStyled;
