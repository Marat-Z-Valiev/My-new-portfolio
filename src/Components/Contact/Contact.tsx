import React from "react";
import ContactStyled from "./ContactStyled";

const Contact: React.FC = () => {
	return (
		<ContactStyled>
			<div id="contact">
				<h1>Contact</h1>
				<div className="links">
					<div className="socialLinks">
						<a
							href="https://www.linkedin.com/in/marat-valiev-89352390/"
							target="blank"
						>
							<i className="fab fa-linkedin fa-3x"></i>
						</a>
					</div>
					<div className="socialLinks">
						<a href="https://github.com/Marat-Z-Valiev" target="blank">
							<i className="fab fa-github-square fa-3x"></i>
						</a>
					</div>
				</div>
			</div>
		</ContactStyled>
	);
};

export default Contact;
