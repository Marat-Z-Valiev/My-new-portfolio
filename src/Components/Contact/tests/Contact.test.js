import React from "react";
import {shallow, configure} from "enzyme";
import toJSON from "enzyme-to-json";
import Adapter from "enzyme-adapter-react-16";
import Contact from "../Contact";

configure({adapter: new Adapter()});

describe("Contact", () => {
	it("should render correctly", () => {
		const wrapper = shallow(<Contact />);
		expect(toJSON(wrapper)).toMatchSnapshot();
	});
});
