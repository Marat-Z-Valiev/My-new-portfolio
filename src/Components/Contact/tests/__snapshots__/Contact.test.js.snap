// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact should render correctly 1`] = `
<ContactStyled__ConactStyled>
  <div
    id="contact"
  >
    <h1>
      Contact
    </h1>
    <div
      className="links"
    >
      <div
        className="socialLinks"
      >
        <a
          href="https://www.linkedin.com/in/marat-valiev-89352390/"
          target="blank"
        >
          <i
            className="fab fa-linkedin fa-3x"
          />
        </a>
      </div>
      <div
        className="socialLinks"
      >
        <a
          href="https://github.com/Marat-Z-Valiev"
          target="blank"
        >
          <i
            className="fab fa-github-square fa-3x"
          />
        </a>
      </div>
    </div>
  </div>
</ContactStyled__ConactStyled>
`;
